esp/public/media/scripts/ajaxschedulingmodule/ESP/SectionInfoPanel.js

```diff
@@ -103,12 +103,21 @@ function SectionInfoPanel(el, sections, togglePanel, sectionCommentDialog) {
 
     }.bind(this);
 
+    var getTeacherLinks = function(section) {
+        var teacher_links_list = []
+            $j.each(section.teacher_data, function(index, teacher) {
+                teacher_links_list.push("<a target='_blank' href=/manage/userview?username=" + encodeURIComponent(teacher.username) + ">" + teacher.first_name + " " + teacher.last_name + "</a>");
```

Comment:
Does the first_name and last_name needed to be HTML-encoded in case it contains special characters like < or >?

Reply:
So I tested this-- as is, if the name has ">" in it, everything looks good, with the character showing up correctly and not breaking any html. Inspecting the html of the element, it looks like it automatically got converted to " &gt;".